docker-compose.yml

```diff
@@ -92,6 +92,10 @@ services:
       - "logging=true"
     restart: always
     logging: *default-logging
+  impersonation:
+    image: lblod/impersonation-service:0.2.0-rc.5
```

Comment:
This needs to be updated to a version that includes the ["transparent impersonation" rework](https://github.com/lblod/impersonation-service/pull/6) before we can merge this.